db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255)
```

Comment:
As task requires:
Each player should choose a race to play, such as `Elf`, `Dwarf`, `Human`, or `Ork`.
You should use the `choices` option to define the possible races for players to select from.